pkg/synthetictests/kubelet.go

```diff
@@ -108,11 +107,17 @@ func testContainerFailures(events monitorapi.Intervals) []*ginkgo.JUnitTestCase
 
 	const failToStartTestName = "[sig-architecture] platform pods should not fail to start"
 	if len(failures) > 0 {
+		qpsMsg := ""
+		for  _, failure := range failures:
```

Comment:
I think you were in Python-mode or something.  [Go needs braces][1]:

```go
for _, failure := range failures {
  if strings.Contains(failure, "pull QPS exceeded") {
    qpsMsg = "..."
    break
  }
}
```

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/26340/pull-ci-openshift-origin-master-images/1417295007534026752#1:build-log.txt%3A69

Reply:
im gonna pretend this didnt happen